### DISCUSSION TITLE
Possible to add more homebrew games to the default feed

### DISCUSSION THREAD

**Discussion by ghamson** [webrcade/webrcade#48] [Ideas]
Hi,

I love this project and with all the new GBStudio games coming out for free on itch.io, this would be a great way to advertise their games as well as your awesome platform.

With that in mind… is there a process to get games reviewed and added to the default feed?

**Comment by raz0red** [comment]
Feel free to just add the games here that you would like to see added, and I can work to add them.
This thread could serve as a location to request them being added.
Thanks!

**Comment by aehlke** [comment]
Are there any Japanese language homebrew games?
